src/core/VstSyncController.cpp

```diff
@@ -25,16 +25,14 @@
 
 #include <QDebug>
 
+#include "VstSyncController.h"
+
 #include "AudioEngine.h"
 #include "ConfigManager.h"
 #include "Engine.h"
-#include "VstSyncController.h"
 #include "RemotePlugin.h"
```

Comment:
IWYU might have done it correct. It's me who added that `#include`. I did this, because by default, IWYU would have included headers for everything that is being used, even if transitively included. So it's human fault :smiley: 

Anyways, will fix in next commit.

Reply:
(Note that clang-format would have fixed the order anyways)